consensus/src/execution_pipeline.rs

```diff
@@ -34,6 +34,10 @@ use std::{
 };
 use tokio::sync::{mpsc, oneshot};
 
+/// Smallest number of transactions Rayon should put into a single worker task.
+/// Same as in execution/executor-benchmark/src/block_preparation.rs
+pub const SIG_VERIFY_RAYON_MIN_THRESHOLD: usize = 32;
```

Comment:
nit: can't we just have one const and share across both files? But way better than a magic number

Reply:
I don't want to import consensus into executor benchmark. we could move those elsewhere, but will leave that for some future thinking.

Reply:
Sounds good!